libbeat/outputs/kafka/config.go

```diff
@@ -37,6 +38,11 @@ import (
 	"github.com/elastic/beats/v7/libbeat/outputs/codec"
 )
 
+type backoffConfig struct {
+	Init time.Duration
```

Comment:
do these need annotations (`config:"init"` etc)?

Reply:
I don't think they _need_ them but I think it's a good idea to be explicit. Added in a0c8093f42c7e36f793e1629f6d8b0372496f644.